### PR TITLE
S3 bucket policies

### PR DESCRIPTION
**Pull request by paulschwarzenberger**
- Mark unencrypted S3 buckets for deletion in 3 days time
- Unmark S3 buckets previously scheduled for deletion which have since been encrypted
- Delete S3 buckets marked for deletion

### CODE REVIEW

File: custodian/policies/ec2/ec2-delete-marked.yml [hunk 1]
```diff
@@ -5,18 +5,19 @@ policies:
       schedule: "rate(1 day)"
       role: arn:aws:iam::{account_id}:role/CustodianDeleteMarkedEC2
     resource: aws.ec2
-    comments: |
+    description: |
       Delete any EC2 instances scheduled for deletion
```
Comment (TomJKing) [inline custodian/policies/ec2/ec2-delete-marked.yml]: Should the description be changed to: "Terminate any EC2 ...", to match the wording of the action descriptions?

File: custodian/scripts/deploy-custodian.sh [hunk 2]
```diff
@@ -6,6 +6,7 @@ COST_CENTRE=$(aws ssm get-parameter --name /mgmt/cost_centre | jq '.Parameter.Va
 SLACK_WEBHOOK=$(aws ssm get-parameter --name /mgmt/slack/webhook | jq '.Parameter.Value' | tr -d '"')
 HOSTED_ZONE=$(aws route53 list-hosted-zones --max-items 1 | jq '.HostedZones[0].Name' | tr -d '"' | sed 's/.$//')
 CUSTODIAN_REGION="eu-west-2"
```
Comment (TomJKing) [inline custodian/scripts/deploy-custodian.sh]: Should this be changes to: CUSTODIAN_REGION_1?

**Review by TomJKing — APPROVED** [review]
Looks OK. Couple of very minor suggestions.

File: custodian/policies/ec2/ec2-delete-marked.yml (hunk 1 above)
Comment (paulschwarzenberger) [reply custodian/policies/ec2/ec2-delete-marked.yml]: thanks, updated as suggested

File: custodian/scripts/deploy-custodian.sh (hunk 2 above)
Comment (paulschwarzenberger) [reply custodian/scripts/deploy-custodian.sh]: thanks, updated as suggested

**Pull request merged**